### PR TITLE
Dr 969 Ui domain fixes

### PR DESCRIPTION
**Pull request by mvillaseco**
Changes:
 - Focus on input when the input it's shown.
 - Fix add domain button height.
 - Highlight the new domain added.

### CODE REVIEW

File: src/wwwroot/controllers/DomainManagerCtrl.js [hunk 1]
```diff
@@ -27,10 +27,11 @@
 
       vm.showNewDomainInput = function(){
         vm.showDomainInput = true;
+        document.getElementById("domain").focus();
```
Comment (andresmoschini) [inline src/wwwroot/controllers/DomainManagerCtrl.js]: Could you take with another angular developer to find an alternative?

File: src/wwwroot/partials/settings/domain-manager.html [hunk 2]
```diff
@@ -8,18 +8,18 @@ <h2 class="width--full">{{'domain-manager_title' | translate}}</h2>
       <button class="button button--large domain--input-toggler" ng-click="vm.showNewDomainInput()">{{'domain-manager_button' | translate}}</button>
     </div>
   </div>
-  <form method="post" id="form" name="form" ng-submit="vm.addDomain(form)" novalidate>
+  <form method="post" id="form" name="form" novalidate>
```
Comment (andresmoschini) [inline src/wwwroot/partials/settings/domain-manager.html]: I liked the call to controller in `form` in place of `button`, why did you do this change?

File: src/wwwroot/partials/settings/domain-manager.html [hunk 3]
```diff
@@ -39,7 +39,7 @@ <h2 class="width--full">{{'domain-manager_title' | translate}}</h2>
           <th class="column-small">{{ 'domain_manager_status_th' | translate}}</th>
           <th class="column-medium">{{ 'domain_manager_actions_th' | translate}}</th>
         </tr>
-        <tr class="column-records" ng-repeat="d in vm.domains" ng-class="d.disabled ? 'grey' : ''">
+        <tr class="column-records" ng-repeat="d in vm.domains" ng-class="{'highlight--temporal-blink' : vm.newDomain == d.name, 'grey' : d.disabled}">
```
Comment (andresmoschini) [inline src/wwwroot/partials/settings/domain-manager.html]: How did you to stop the blinking?

Comment (andresmoschini) [inline src/wwwroot/partials/settings/domain-manager.html]: Oh, with CSS, sorry

Comment (mvillaseco) [inline src/wwwroot/partials/settings/domain-manager.html]: By using "animation-iteration-count: 5;" the animation(blink) iterates 5 times.

**Comment by mvillaseco** [comment]
Yes, we can pass the domain value to the controller by adding into the scope, just like in registration or in login, i'll do it.

**Pull request merged**